p2p_wallet/Scenes/Main/ReceiveFundsViaLink/ClaimSentViaLinkTransaction.swift

```diff
@@ -2,15 +2,29 @@ import Foundation
 import SolanaSwift
 import Send
 import Resolver
+import FeeRelayerSwift
 
 struct ClaimSentViaLinkTransaction: RawTransactionType {
+    // MARK: - Nested type
+    
+    enum FakeTransactionErrorType: String, CaseIterable, Identifiable {
+        case noError
+        case networkError
+        case otherError
+        var id: Self { self }
+    }
+    
+    // MARK: - Properties
+
     let claimableTokenInfo: ClaimableTokenInfo
     let token: Token
     let destinationWallet: Wallet
     let tokenAmount: Double
     
     let payingFeeWallet: Wallet? = nil
     let feeAmount: FeeAmount = .zero
+    let isFakeTransaction: Bool
```

Comment:
is it possible to separate FakeTransaction from the ordinary one? Make FakeClaimSentViaLinkTransaction for example?